internal/cloudapi/v2/v2_test.go

```diff
@@ -2322,18 +2427,20 @@ func TestComposeRequestMetadata(t *testing.T) {
 		"status": "pending"
 	}`, jobId, jobId))
 
-	// metadata response should include the ComposeRequest even when build is not done
-	test.TestRoute(t, srv.Handler("/api/image-builder-composer/v2"), false, "GET", fmt.Sprintf("/api/image-builder-composer/v2/composes/%v/metadata", jobId), ``, http.StatusOK,
-		fmt.Sprintf(`{
+			// metadata response should include the ComposeRequest even when build is not done
```

Comment:
Nitpick: it would be still good to test the endpoint with the compose being done for the reasons I mentioned in https://github.com/osbuild/osbuild-composer/pull/4904#discussion_r2519430439. But not a blocker from my PoV, because the original issue that would manifest in such a scenario has been fixed.